plugins/epos/plugin.py

```diff
@@ -1025,6 +1032,98 @@ def rda_a1_1_01d(self):
 
         return (points, msg_list)
 
+    @ConfigTerms(term_id="terms_vocabularies")
+    def rda_i1_01m(self, **kwargs):
+        """Indicator RDA-I1-01M: Metadata uses knowledge representation expressed in standarised format.
+
+        This indicator is linked to the following principle: I1: (Meta)data use a formal,
+        accessible, shared, and broadly applicable language for knowledge representation.
+
+        The indicator serves to determine that an appropriate standard is used to express
+        knowledge, in particular the data model and format.
+
+        Returns
+        -------
+        points
+            100/100 If the file format is listed under IANA Internet Media Types
```

Comment:
Not only media types is being checked here

Reply:
Corrected